projects/packages/post-list/composer.json

```diff
@@ -0,0 +1,53 @@
+{
+	"name": "automattic/jetpack-post-list",
+	"description": "PostList",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
```

Comment:
Since we rely on the Assets package, let's set it up as a dependency here.

Reply:
Added here: https://github.com/Automattic/jetpack/pull/20720/commits/1a0d63a6e815af958faf0e9b012bab9672773663